### PR TITLE
adding qubekit.

### PR DESCRIPTION
**Pull request by jthorton**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jthorton** [comment]
@CRingrose94 please comment here to confirm you would like to be a maintainer as well.

**Comment by CRingrose94** [comment]
> @CRingrose94 please comment here to confirm you would like to be a maintainer as well.

I would like to be a maintainer too!

File: recipes/qubekit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+source:
+  url: https://github.com/qubekit/QUBEKit/archive/v.{{ version }}.tar.gz
+  sha256: 61b940d34e74315da58fed624a134a4d39454c2cf21059b01b00f2aa71c0fed8
+
+
+build:
+  number: 0
+  skip: True  # [win or py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
```
[the remark this answers is not in the document]
Comment (jthorton) [reply recipes/qubekit/meta.yaml]: I think you're right I have changed this, but it looks like one of our dependencies does not have windows version and the checks are now failing.

File: recipes/qubekit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - QUBEKit-gui = QUBEKit.GUI.gui:main
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
```
Comment (scopatz) [inline recipes/qubekit/meta.yaml]: You don't need steuptools if you have pip

File: recipes/qubekit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  license_file: LICENSE.txt
+  summary: Quantum mechanical bespoke force field derivation toolkit
+  doc_url: https://github.com/qubekit/QUBEKit
+  description: QUBEKit is a Python based force field parameter derivation toolkit. Our aims are to allow users to quickly derive molecular mechanics parameters directly from quantum mechanical calculations.
+  dev_url: https://github.com/qubekit/QUBEKit
+
+extra:
+  recipe-maintainers:
+    - jthorton
+    - CRingrose94
```
Comment (scopatz) [inline recipes/qubekit/meta.yaml]: @CRingrose94 are you OK with being a maintainer

File: recipes/qubekit/meta.yaml (hunk 2 above)
Comment (jthorton) [reply recipes/qubekit/meta.yaml]: thanks removed!

File: recipes/qubekit/meta.yaml (hunk 3 above)
Comment (jthorton) [reply recipes/qubekit/meta.yaml]: see above ^
> > @CRingrose94 please comment here to confirm you would like to be a maintainer as well.
> 
> I would like to be a maintainer too!

Comment (CRingrose94) [reply recipes/qubekit/meta.yaml]: @scopatz Yes I'd like to be a maintainer please.

File: recipes/qubekit/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (scopatz) [reply recipes/qubekit/meta.yaml]: Only the linux builds matter for noarch

**Comment by jthorton** [comment]
Thanks for the help :+1:

**Comment by scopatz** [comment]
:+1:

**Pull request merged**